packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -77,13 +77,12 @@ describe('HCP home page', () => {
     })
 
     await checkHCPLayout(`${firstName} ${lastName}`, { teamName: teamThree.name }, AVAILABLE_TEAMS)
-    checkSecondaryBar(false, true)
+    checkPatientList()
 
     expect(screen.queryAllByLabelText('flag-icon-active')).toHaveLength(0)
     expect(screen.getAllByLabelText('flag-icon-inactive')).toHaveLength(4)
 
-    const patientRow = screen.queryByTestId(`patient-row-${unmonitoredPatient.userid}`)
-    const removeButton = within(patientRow).getByRole('button', { name: `Remove patient ${unmonitoredPatient.profile.email}`, hidden: true })
+    const removeButton = screen.getByRole('button', { name: `Remove patient ${unmonitoredPatient.userid}` })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I feel like this test is missing the opportunity to test the full patient list. What about selecting it fully having a `toHaveTextContent`?